File: Server/BluetoothServer/BluetoothServer/src/main/java/ApplicationModels/Temperature.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ApplicationModels;

/**
 *
 * @author Jarno
 */
public class Temperature
{
    private String sensorname;
    private double value;
    private long timestamp;
    private TemperatureUnit unit;

    public void setTimestamp(long timestamp)
    {
        this.timestamp = timestamp;
    }
    
    
    
    public static class TemperatureUnit
    {
        private String suffix;
        private String unit;
    }
}

File: WebInterface/src/app/database/temperature.ts
export class Temperature {

  id: string;
  sensorname: string;
  value: number;
  timestamp: number;



  constructor(id: string, sensorname: string, value: number, timestamp: number) {
    this.id = id;
    this.sensorname = sensorname;
    this.value = value;
    this.timestamp = timestamp;
  }

}
